### PR TITLE
Feature : Login/Register, Avatar

### PR DESCRIPTION
**Pull request by SalonSikdar**
Added Login feature
Added redirect to Register Feature
Scripted Avatar Component

### CODE REVIEW

File: css/night-owl.css [hunk 1]
```diff
@@ -1,135 +1,173 @@
...
-    font-style: italic;
+  color: #637777;
+  font-style: italic;
+  background: #011627;
 }
 .hljs-doctag {
-    color: #7fdbca;
+  color: #7fdbca;
+  background: #011627;
 }
```
Comment (Sahil25061999) [inline css/night-owl.css]: variables can be used for color and background

File: pages/components/Avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+.avatar-xl img {
+  width: 12em;
+  height: 12em;
+}
+
+.avatar-lg img {
+  width: 10em;
+  height: 10em;
+}
```
Comment (Sahil25061999) [inline pages/components/Avatar/avatar.css]: use rem if possible

File: css/night-owl.css (hunk 1 above)
Comment (SalonSikdar) [reply css/night-owl.css]: Ack, Thanks for the suggestion, will implement

**Review by archit7448 — APPROVED** [review]
you can use rems and image size is big take time to load everything looks good

File: pages/components/Avatar/avatar.css (hunk 2 above)
Comment (SalonSikdar) [reply pages/components/Avatar/avatar.css]: Yes, done. Thanks

**Comment by SalonSikdar** [comment]
> you can use rems and image size is big take time to load everything looks good

Done

File: pages/components/Avatar/avatar.css (hunk 2 above)
Comment (Sahil25061999) [reply pages/components/Avatar/avatar.css]: looks good can be be merged

**Pull request merged**